packages/algob/test/internal/core/config/config-loading.ts

```diff
@@ -205,14 +205,15 @@ describe("config loading", function () {
     });
 
     it("should detect conflict of account names", async function () {
-      const spy = sinon.spy(console, 'warn');
+      // console is mocked in package.json mocha options
+      let stub = console.warn as sinon.SinonStub;
```

Comment:
curious - what's the difference between using `spy` and `stub`? Is using `stub` better ?

Reply:
I was usually using stubs, but it seams it's better to use spy here. I will update to spy. Thanks for asking.
here is an overview: https://www.javatpoint.com/mock-vs-stub-vs-spy